src/main/java/com/automation/school/ad_main/Collection/CompanyOrder.java

```diff
@@ -0,0 +1,87 @@
+package com.automation.school.ad_main.Collection;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+public class CompanyOrder implements Order{
+
+
+    private long id; //(generate automatically in constructor )
+    private LocalDateTime date; //(set as current day directly in constructor)
+    private User user;
+    public List<Company> companies;
+    private static int nextUniqueId = 0;
+    private int itemCount = 0;
+    Double value;
+
+    // public  User newUser;
+    //public  Item newItem;
+    //get the value of id
+    public CompanyOrder() {
+        this.date = LocalDateTime.now();
+        this.companies = new ArrayList<>(itemCount);
+        this.value = value;
+    }
+
+
+
+
+    @Override
+    public double sum() {
+        return 0;
+    }
+
+    @Override
+    public int count() {
+        return 0;
+    }
+
+    @Override
+    public LocalDateTime date() {
+        return null;
+    }
+
+    @Override
+    public void registrationUser(User user) {
+
+    }
+
+    @Override
+    public void addItem(Item items) {
+
+    }
+
+    public double sum(double price) {
```

Comment:
could you please explain, what are you trying to achieve here?
as well as for method count(Item item) on line 65?

Reply:
1) Achievement was added. public void addItem(Item items) { this.itemsUser.add(items)}
2) It was row method to start count items in array was deleted.